python/cudf/utils/ioutils.py

```diff
@@ -143,12 +143,14 @@
 
 Parameters
 ----------
-path_or_buf : a valid JSON string or file-like, default: None
-    The string could be a URL. Valid URL schemes include http, ftp, s3,
-    gcs, and file. For file URLs, a host is expected. For instance, a local
-    file could be ``file://localhost/path/to/table.json``
+path_or_buf : a valid JSON string or file-like
+    The string could be a URL (pyarrow engine only). Valid URL schemes include
+    http, ftp, s3, gcs, and file. For file URLs, a host is expected.
+    For instance, a local file could be ``file://localhost/path/to/table.json``
+    engine : {{ 'cudf', 'pyarrow' }}, default 'cudf'
```

Comment:
```suggestion
    engine : {{ 'cudf', 'pandas' }}, default 'cudf'
```
The current backend for read and write JSON is actually using PANDAS...
All the other places that read `pyarrow engine only` should be `pandas engine only` or similar.

Reply:
fixed, thanks